apps/hello-world-vuejs/src/App.vue

```diff
@@ -0,0 +1,56 @@
+<template>
+  <div id="app">
+    <div>Hello {{ user.name }}</div>
+    <div>Your context: {{ JSON.stringify(context) }}</div>
+    <div>Your settings: {{ JSON.stringify(settings) }}</div>
+  </div>
+</template>
+
+<script>
+import mondaySdk from "monday-sdk-js";
+const monday = mondaySdk();
+
+export default {
+  name: "app",
+  data() {
+    return {
+      settings: "",
+      context: "",
+      user: ""
+    };
+  },
+
+  created() {
+    monday.listen("settings", this.getSettings);
+    monday.listen("context", this.getContext);
+    this.fetchUserName();
```

Comment:
i think that it should be:
```
monday.api(`query { me { name } }`).then(this.getUser);
```

and add a method bellow that will be identical to the "getSettings"